cmis/src/main/java/org/frankframework/extensions/cmis/CmisSender.java

```diff
@@ -270,47 +277,39 @@ public void configure() throws ConfigurationException {
 	 * Creates a session during JMV runtime, tries to retrieve parameters and falls back on the defaults when they can't be found
 	 */
 	public CloseableCmisSession createCmisSession(ParameterValueList pvl) throws SenderException {
-		String authAlias_work = null;
-		String username_work = null;
-		String password_work = null;
+		String cfAuthAlias = getParameterOverriddenAttributeValue(pvl, "authAlias", getAuthAlias());
+		String cfUsername = getParameterOverriddenAttributeValue(pvl, "username", getUsername());
+		String cdPassword = getParameterOverriddenAttributeValue(pvl, "password", getPassword());
 
-		if (pvl != null) {
-			ParameterValue pv = pvl.get("authAlias");
-			if (pv != null) {
-				authAlias_work = pv.asStringValue();
-			}
-			pv = pvl.get("username");
-			if (pv == null) {
-				pv = pvl.get("userName");
-			}
-			if (pv != null) {
-				username_work = pv.asStringValue();
-			}
-			pv = pvl.get("password");
-			if (pv != null) {
-				password_work = pv.asStringValue();
+		CredentialFactory cf = new CredentialFactory(cfAuthAlias, cfUsername, cdPassword);
+		try {
+			Map<String, String> headers = new HashMap<>();
+			if(pvl != null) {
+				for(Entry<String, Object> entry : pvl.getValueMap().entrySet()) {
```

Comment:
De `ParameterValue` class die in the `ParameterValueList` zit kan ook de parameter value al als string terug geven, waarom gebruik je die eigenlijk niet?